src/Microsoft.Security.Utilities.Core/WellKnownRegexPatterns.cs

```diff
@@ -17,95 +19,66 @@ public static class WellKnownRegexPatterns
+                   new LooseSasSecret(),
+                   new OAuth2BearerToken(),
+                   new Unclassified32ByteBase64String(),
+                   new Unclassified64ByteBase64String(),
+                   new AadClientAppLegacyCredentials34(),      // SEC101/101 legacy generated passwords.
+                   new Pkcs12CertificatePrivateKeyBundle(),
+                   new Unclassified16ByteHexadecimalString()
+    );
+
+    public static IReadOnlyList<RegexPattern> HighConfidenceMicrosoftSecurityModels { get; } =
+        CreateList(new UnclassifiedLegacyCommonAnnotatedSecurityKey(),
+                   new AadClientAppIdentifiableCredentials(),
+                   new AzureFunctionIdentifiableKey(),
+                   new AzureSearchIdentifiableQueryKey(),
+                   new AzureSearchIdentifiableAdminKey(),
+                   new AzureRelayIdentifiableKey(),
+                   new AzureEventHubIdentifiableKey(),
+                   new AzureServiceBusIdentifiableKey(),
+                   new AzureIotHubIdentifiableKey(),
+                   new AzureIotDeviceIdentifiableKey(),
+                   new AzureIotDeviceProvisioningIdentifiableKey(),
+                   new AzureStorageAccountIdentifiableKey(),
+                   new AzureCosmosDBIdentifiableKey(),
+                   new AzureBatchIdentifiableKey(),
+                   new AzureMLWebServiceClassicIdentifiableKey(),
+                   new AzureApimIdentifiableDirectManagementKey(),
+                   new AzureApimIdentifiableSubscriptionKey(),
+                   new AzureApimIdentifiableGatewayKey(),
+                   new AzureApimIdentifiableRepositoryKey(),
+                   new AzureCacheForRedisIdentifiableKey(),
+                   new AzureContainerRegistryIdentifiableKey(),
+                   new NuGetApiKey(),
+                   new AdoLegacyPat(),                         // SEC101/102
+                   new AzureCosmosDBLegacyCredentials(),       // SEC101/104
+                   new AzureStorageAccountLegacyCredentials(), // SEC101/106
+                   new AzureMessagingLegacyCredentials(),
+                   new AzureDatabricksPat(),
+                   new AzureEventGridIdentifiableKey());
+
+    public static IReadOnlyList<RegexPattern> DataClassification { get; } =
+        CreateList(new IPv4(),
+                   new IPv6(),
+                   new Float(),
+                   new Integer(),
+                   new GuidValue());
+
+    public static IReadOnlyList<RegexPattern> HighConfidenceThirdPartySecurityModels { get; } =
+        CreateList(new NpmAuthorKey(),
+                   new SecretScanningSampleToken());
+
+    public static IReadOnlyList<RegexPattern> PreciselyClassifiedSecurityKeys { get; } =
+        CombineLists(HighConfidenceMicrosoftSecurityModels,
+                     HighConfidenceThirdPartySecurityModels);
+
+    public static IReadOnlyList<RegexPattern> SecretStoreClassificationDetections { get; } =
```

Comment:
>SecretStoreClassificationDetections [](http://example.com/codeflow?start=46&length=35)

this is intended capture the rules we would want to run when enumerating a secret store, but the idea is unrealized and unconsumed, lets zap it for now.

Reply:
#196